lisa/tools/ntp.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import re
+from typing import cast
+
+from retry import retry
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+from lisa.util import LisaException
+
+from .service import Service
+
+
+class Ntp(Tool):
+    __leap_pattern = re.compile(r"([\w\W]*?)leap_none.*$", re.MULTILINE)
+
+    @property
+    def command(self) -> str:
+        return "ntpq"
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def install(self) -> bool:
+        posix_os: Posix = cast(Posix, self.node.os)
+        package_name = "ntp"
+        posix_os.install_packages(package_name)
+        return self._check_exists()
+
+    @retry(tries=20, delay=1)
+    def check_leap_code(self) -> None:
+        cmd_result = self.run("-c rv", shell=True, sudo=True, force_run=True)
+        if not self.__leap_pattern.match(cmd_result.stdout):
+            raise LisaException("The leap code is not lean_none.")
```

Comment:
What happens, if the lead code is not lean_none? Add more explanation in the exception.